grammar.js

```diff
@@ -853,7 +846,6 @@ module.exports = grammar({
         $.range_pattern,
         $.exception_pattern
       ),
-      optional($.type_annotation),
```

Comment:
It was removed because a let binding was created two `field` patterns.

Example:

```res
let a: int
```

```
let_declaration [0, 0] - [0, 10]
  let_binding [0, 4] - [0, 10]
    pattern: value_identifier [0, 4] - [0, 5]
    pattern: type_annotation [0, 5] - [0, 10]
      type_identifier [0, 7] - [0, 10]
```